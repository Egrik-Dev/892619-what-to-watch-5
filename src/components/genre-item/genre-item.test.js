import React from 'react';
import renderer from 'react-test-renderer';
import GenreItem from './genre-item';

const genre = [`ALL_GENRES`, `All genres`];

it(`Should GenreItem render correctly if it is not an active genre`, () => {
  const tree = renderer
    .create(<GenreItem
      isActiveGenre={false}
      genre={genre}
      onFilterChange={() => {}}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Should GenreItem render correctly if it is an active genre`, () => {
  const tree = renderer
    .create(<GenreItem
      isActiveGenre={true}
      genre={genre}
      onFilterChange={() => {}}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
