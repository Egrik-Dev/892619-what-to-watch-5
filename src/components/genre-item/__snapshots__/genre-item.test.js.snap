// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should GenreItem render correctly if it is an active genre 1`] = `
<li
  className="catalog__genres-item catalog__genres-item--active"
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    All genres
  </a>
</li>
`;

exports[`Should GenreItem render correctly if it is not an active genre 1`] = `
<li
  className="catalog__genres-item "
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    All genres
  </a>
</li>
`;
